pagopa/io-web-profile
# [IOPID-1007] : Importing Accordion from Editorial Components

## Short description

The purpose of this PR is to replace the current custom accordion, which natively uses the Material UI accordion, with the one deployed on [pagopa-editorial-components](https://pagopa.github.io/pagopa-editorial-components). This replacement already reflects the desired style and layout and will handle future changes from the common library. We noticed these differences during the UAT. When the project started, we couldn't find this component on MUI-Italia.

Please refer to the [ticket](https://pagopa.atlassian.net/browse/IOPID-1007?atlOrigin=eyJpIjoiYmU2ZWQ0YWY5MTRkNDJjZDg2ZWZkNmIxMmI2YTZkZTUiLCJwIjoiaiJ9)

## List of changes proposed in this pull request

- The custom accordion has been removed
- Import the Accordion from pagopa-editorial-components

## How to test

Launch the project and verify that the component is imported correctly, for example on the page http://localhost:3000/it/esci/.
